### PR TITLE
added einnovator cloud-manager stack based on chart einnovator-devops

### PR DESCRIPTION
**Pull request by jsimao71**
## BACKGROUND
This is a pull request to add EInnovator Cloud Manager web tool to Digital Ocean Kubernetes marketplace.
Cloud Manager is a compressive UI/IDE for Kubernetes, to be used as a more advanced replacement to Kubernetes Dashboard and complement to command-line tools like kubectl.
The folder stacks/cloud-manager was added, with deploy.sh script for performing the install.
Generated using the instructions specified in this repo, and reported to us by: Daniel Levy <dlevy@digitalocean.com>.
Uses chart einnovator-devops, pulled from helm repo located in:  https://cdn.einnovator.org/charts
Installation is done in namespace "einnovator". App version is latest.
With the default chart settings install is done with minimal resource footprint. Namely, deploys the tool in singleuser mode (no need for side-kick SSO Gateway), with a single docker image, and uses an embeded db rather than deploying a mysql/mariadb on the side. Tool is setup in free to use mode.

For a quick overview of the tool features you can use the youtube video and companion article "Kubernetes Just Got Easy" in the links below:
Youtube :
https://www.youtube.com/watch?v=ZT8xMFwjYd4
Article:
https://cms.einnovator.org/publication/kubernetes-just-got-easy

-----------------------------------------------------------------------

## Changes
* List the changes that you made

-----------------------------------------------------------------------

## Checklist
- [ ] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/cloud-manager/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  # use local version of values.yml
+  ROOT_DIR=$(git rev-parse --show-toplevel)
+  values="$ROOT_DIR/stacks/cloud-manager/values.yml"
+else
+  # use github hosted master version of values.yml
+  values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/cloud-manager/values.yml"
+fi
+
+helm upgrade "$STACK" "$CHART" \
+  --atomic \
```
Comment (paulanunda) [inline stacks/cloud-manager/deploy.sh:29]: Please remove the extra `--atomic`

File: stacks/cloud-manager/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add ei https://cdn.einnovator.org/charts
```
Comment (paulanunda) [inline stacks/cloud-manager/deploy.sh:8]: When I attempt to run this, I get the following:
```
Error: looks like "https://cdn.einnovator.org/charts" is not a valid chart repository or cannot be reached: Get "https://cdn.einnovator.org/charts/index.yaml": dial tcp: lookup cdn.einnovator.org: no such host
```

File: stacks/cloud-manager/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```
Comment (paulanunda) [inline stacks/cloud-manager/deploy.sh:1]: Please make this file an executable. `chmod +x`

**Review by paulanunda — CHANGES_REQUESTED** [review]

File: stacks/cloud-manager/deploy.sh (hunk 2 above)
Comment (jsimao71) [reply stacks/cloud-manager/deploy.sh:8]: Hi, @paulanunda !
I just run the command both from my windows and from my mac and it worked.
helm repo add ei https://cdn.einnovator.org/charts
outputs:
"ei" has been added to your repositories
Can you please try again..
The file your error message is complaining is there:
https://cdn.einnovator.org/charts/index.yaml
Copy this URL on the browser and do a curl from the machine you are running to double check.
Please let me know if the issue still persists.
Thanks,
Jorge.

File: stacks/cloud-manager/deploy.sh (hunk 1 above)
Comment (jsimao71) [reply stacks/cloud-manager/deploy.sh:29]: @paulanunda submited second PR, with the --atomic removed..please check..
Please try again the issue of the:
 https://cdn.einnovator.org/charts
Let me know if it is reachable for your machine now..
Thanks,
J.

File: stacks/cloud-manager/deploy.sh (hunk 3 above)
Comment (jsimao71) [reply stacks/cloud-manager/deploy.sh:1]: Fixed this as a second PR (3-2). (Ignore PR3...no need to review)
Please let me know if this fixes work for you, and you manage to deploy the app.
Thanks,
J.

File: stacks/cloud-manager/deploy.sh (hunk 2 above)
Comment (jsimao71) [reply stacks/cloud-manager/deploy.sh:8]: Also try:
ping cdn.einnovator.org
to see if it is a IP connectivity problem.
Thanks!

**Comment by paulanunda** [comment]
Closing in favor of #180

**Pull request closed without merging**